package.json

```diff
@@ -27,7 +27,8 @@
     "react-dom": "^18.2.0",
     "react-icons": "^4.4.0",
     "styled-components": "^5.3.5",
-    "tailwindcss": "^3.1.6"
+    "tailwindcss": "^3.1.6",
+    "yarn": "^1.22.19"
```

Comment:
We should not have yarn locally on the project. You may have run `npm i yarn`. The correct command is `npm i yarn -g` which installs yarn on your machine globally, not on the project.

Reply:
The yarn was uninstalled with sucess. Thank you.